src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIPacket.cs

```diff
@@ -146,12 +176,18 @@ public void GetData(byte[] buffer, ref int dataSize)
         /// </summary>
         /// <param name="data">Data</param>
         /// <param name="length">Length</param>
-        public void SetData(byte[] data, int length)
+        public void SetData(byte[] data, int length, int capaticy = -1, bool isArrayFromArrayPool = false)
```

Comment:
Type: int capacity = -1

Reply:
The SetData below, which is unused seems like the only method calling this overload with isArrayFromArrayPool. If the method below is unused, the method parameter `isArrayFromArrayPool` can be removed.

Reply:
Also the parameter int capacity can be removed since no other method passes in these parameters